src/pipeline/pipeline-preview.ts

```diff
@@ -44,12 +44,12 @@ export async function showPipelinePreview(): Promise<void> {
   }
 }
 
-export async function showPipelineRunPreview(name: string): Promise<void> {
+export async function showPipelineRunPreview(name: string, uid?: string): Promise<void> {
   if (!name) {
     return;
   }
-  const uri = tektonFSUri(ContextType.PIPELINERUN, name, 'yaml');
-  const pipelineRunDoc = await tektonVfsProvider.loadTektonDocument(uri);
+  const uri = tektonFSUri(ContextType.PIPELINERUN, name, 'yaml', uid);
+  const pipelineRunDoc = await tektonVfsProvider.loadTektonDocument(uri, (uid) ? false : true);
```

Comment:
You can use `!!` operator instead of `(uid) ? false : true` i.e. : `tektonVfsProvider.loadTektonDocument(uri, !!uid);`

Reply:
Thanks 👍